packit.spec

```diff
@@ -37,8 +37,6 @@ Requires:       python3-koji
 Requires:       python3-bodhi-client
 Requires:       python3-%{real_name} = %{version}-%{release}
 
-%?python_enable_dependency_generator
```

Comment:
What is the reason for removing this?

Reply:
It's enabled by default in all current fedora releases.